config/kme_config.toml

```diff
@@ -38,17 +38,20 @@ ias_url = "https://api.trustedservices.intel.com/sgx/dev"
 ias_api_key = "112233445566778899aabbccddeeff00"
 
 # TEE enclave library to use.
-enclave_library = ""
-enclave_library_path = ""
+enclave_library = "libtcf-enclave.signed.so"
+enclave_library_path = "tc/sgx/trusted_worker_manager/enclave/build/lib/"
 
 num_of_enclaves = "1"
 # MRENCLAVE for WPE that will be allowed to register with this KME
-wpe_mrenclave = ""
+wpe_mrenclave = "CFF2E89D19A02886C653FD77B5D8314414D9C486155013CC0505671B894D053F"
+
+# The type of enclave viz - KME/WPE/SWE
+enclave_type = "KME"
 
 [KMEListener]
 # URL of KMEListener where a Work order processor can send requests meant for
 # the Key Management Enclave(KME).
-bind = "http://localhost:1948" 
+bind = "http://avalon-kme:1948"
```

Comment:
If you do this, you need to add instructions to BUILD.md to edit file kme_config.toml so it will work in standalone mode.
That is, add instructions to change this line to http://localhost:1948

Reply:
Worker pool is an additional feature @danintel . The documentations will have quite a bit of changes. As of now this is not breaking the Singleton(default) flow. Planning to take up all those changes closer to feature completion.